bugsnag-plugin-android-ndk/src/main/jni/featureflags.c

```diff
@@ -0,0 +1,132 @@
+#include <string.h>
+
+#include "bugsnag_ndk.h"
+#include "featureflags.h"
```

Comment:
I'm not 100% sure on how feature flags will be persisted so that they can be read on the next application launch for unhandled NDK errors - would it be possible to get an explanation of how this works?